airflow/cli/commands/dag_command.py

```diff
@@ -378,6 +380,48 @@ def dag_list_dag_runs(args, dag=None):
     print(table)
 
 
+@cli_utils.action_logging
+def generate_pod_yaml(args):
+    """Generates yaml files for each task in the DAG. Used for testing output of KubernetesExecutor"""
+
+    from kubernetes.client.api_client import ApiClient
```

Comment:
This is a command that is intended for the end user. Do you think it's worth adding a friendly message that will be displayed when Kubernetes is not installed or not configured?

Reply:
We have a similar check for Celery.
<img width="1247" alt="Screenshot 2020-09-03 at 23 10 08" src="https://user-images.githubusercontent.com/12058428/92167409-a14e5280-ee3a-11ea-85f0-6f7059b9a994.png">

Reply:
@mik-laj good idea will implement.